### PR TITLE
SAIC-235 Fix condensation for flavors with non-int IDs

### PR DESCRIPTION
**Pull request by roman-verchikov**
Condensation used to fail in case flavor was non-int value. This
patch fixes it.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSED.

File: condensation/flavor.py [hunk 1]
```diff
@@ -18,8 +18,7 @@ class Flavor(object):
     """This class is representation of OpenStack flavor"""
 
     def __init__(self, fl_id, name, ram, core, ephemeral=None, swap=None):
-        # by some reason - id of flavor is string
-        self.fl_id = int(fl_id)
```
Comment (miarmak) [inline condensation/flavor.py]: Don't you know any reason to have it in `int` type before?

Comment (roman-verchikov) [inline condensation/flavor.py]: There's no reason, all openstack objects define IDs as string.

Comment (miarmak) [inline condensation/flavor.py]: Okay. Just trying to find out the reason of this type conversion)

**Comment by miarmak** [comment]
Looks good! +1

**Comment by Wasapl** [comment]
Merged build finished. Test PASSED.

**Pull request merged**